core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -597,6 +599,24 @@ public static String getSegmentIdFromPath(String dataFileAbsolutePath) {
     }
   }
 
+  /**
+   * gets segment id from given absolute segment file path
+   */
+  public static String getSegmentIdFromSegmentFilePath(String segmentFilePath) {
```

Comment:
If used only in test files, can move it to TestUtil

Reply:
Done